lsst/ts/tunablelaser/cli.py

```diff
@@ -1,31 +1,40 @@
 import argh
```

Comment:
Can you add some description about this file? Not sure what is it for. I can try to guess it but a doc would be nice!

Reply:
Do you mean the module or the command line interface?

Reply:
Whatever you think is best.